web/src/containers/Admin/General/index.css

```diff
@@ -161,37 +161,37 @@
 	margin: auto;
 	margin-top: 7px;
 }
-.sell {
+.general-wrapper .sell {
 	border: 1px solid #ffffff;
 	width: 60px;
 	height: 18px;
 }
-.sell .label {
+.general-wrapper .sell .label {
```

Comment:
These classes are overriding the kit styles. It is better to use either a BEM structure or nested classes to avoid this in the future. @jsmitrah